eStore/Core/UI/Theme.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Theme.swift
+//  eStore
+//
+//  Created by Vladislav Kondrashkov on 4/8/19.
+//  Copyright © 2019 Vladislav Kondrashkov. All rights reserved.
+//
+
+import UIKit
+
+class Theme {
```

Comment:
Why isn't it just a protocol? Is there any reason to have a base class? Maybe it would be better to have "ClearTheme" if you need such, otherwise I would prefer to explicitly set all needed colors for the theme and don't rely on deriving some of them from base class.